### PR TITLE
DTGB-536: Added table mobile JS to gent_base.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: source/js/table-mobile.bindings.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * @file
+* Table component binding.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseLoadMenu = {
+    attach: function (context, settings) {
+
+      console.log('test'); // eslint-disable-line no-console
```
Comment (hstandaert) [inline source/js/table-mobile.bindings.js]:
```suggestion
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: source/js/table.bindings.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      var tables = document.querySelectorAll('table');
+      for (var item = 0; item < tables.length; item++) {
+        var tableItem = tables[item];
+
+        // Create wrapper div and wrap it around each table
+        var tableWrapper = document.createElement('div');
+        tableWrapper.classList.add('responsive-table');
+
+        tableItem.parentNode.insertBefore(tableWrapper, tableItem);
+        tableWrapper.appendChild(tableItem);
```
Comment (hstandaert) [inline source/js/table.bindings.js:23]: There's no need to loop over the tables 3 times. You can wrap all this into 1 `for` loop, where you:

- Add the wrapper around the `table` element.
- Create the `ResponsiveTable` object.
- Create the `MobileTable` object.

File: source/js/table.bindings.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      for (var i = 0; i < tablesNodeList.length; i++) {
+        var table = tablesNodeList[i];
+
+        // Adds accessibility support.
+        new ResponsiveTable(table, {
+          scrollableText: '(scroll to see more)'
+        });
+      }
+
+      var selected = document.querySelectorAll('.responsive-table');
```
Comment (hstandaert) [inline source/js/table.bindings.js]: Duplicate selection of `.responsive-table`. It's already stored in `tablesNodeList`.

File: source/js/table.bindings.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      for (var j = selected.length; j--;) {
+        new MobileTable(selected[j]); // eslint-disable-line no-undef
```
Comment (hstandaert) [inline source/js/table.bindings.js]:
```suggestion
        new MobileTable(selected[j]);
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: source/js/table.bindings.js (hunk 2 above)
Comment (meirege) [reply source/js/table.bindings.js:23]: Fixed.

File: source/js/table.bindings.js (hunk 3 above)
Comment (meirege) [reply source/js/table.bindings.js]: Fixed.

File: source/js/table.bindings.js (hunk 4 above)
Comment (meirege) [reply source/js/table.bindings.js]: Fixed.

File: source/js/table.bindings.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        // Create wrapper div and wrap it around each table
+        var tableWrapper = document.createElement('div');
+        tableWrapper.classList.add('responsive-table');
+
+        tableItem.parentNode.insertBefore(tableWrapper, tableItem);
+        tableWrapper.appendChild(tableItem);
+
+        // Adds accessibility support.
+        new ResponsiveTable(tableWrapper, {
+          scrollableText: '(scroll to see more)'
```
Comment (hstandaert) [inline source/js/table.bindings.js]:
```suggestion
          scrollableText: Drupal.t('(scroll to see more)')
```

And you should also add a translation to the `.po` file. 

_P.S. Didn't notice this in my last review, apologies 😇_

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: source/js/table.bindings.js (hunk 5 above)
Comment (meirege) [reply source/js/table.bindings.js]: Done :P

File: translations/nl.po [hunk 6]
```diff
@@ -476,3 +475,6 @@ msgstr "Extra informatie"
 #: themes/contrib/gent_base/templates/custom/vesta/vesta.html.twig:153
 msgid "View all locations from"
 msgstr "Bekijk alle locaties van"
+
+#~ msgid "@name - openinghours and addresses"
```
Comment (hstandaert) [inline translations/nl.po]:
```suggestion
```

File: translations/nl.po [hunk 7]
```diff
@@ -476,3 +475,6 @@ msgstr "Extra informatie"
...
+#~ msgstr "@name - Openingsuren en adressen"
```
Comment (hstandaert) [inline translations/nl.po]:
```suggestion
```

**Review by hstandaert — CHANGES_REQUESTED** [review]

**Review by hstandaert — APPROVED** [review]

**Pull request merged**